components/datadog/agent/ecsFargate.go

```diff
@@ -11,7 +11,7 @@ import (
 
 func ECSFargateLinuxContainerDefinition(e config.Env, image string, apiKeySSMParamName pulumi.StringInput, fakeintake *fakeintake.Fakeintake, logConfig ecs.TaskDefinitionLogConfigurationPtrInput) *ecs.TaskDefinitionContainerDefinitionArgs {
 	if image == "" {
-		image = dockerAgentFullImagePath(e, "public.ecr.aws/datadog/agent", "latest", false)
+		image = dockerAgentFullImagePath(e, "public.ecr.aws/datadog/agent", "", false)
```

Comment:
I expect this to be safe because the only caller of this function is the one from `resources/aws/ecs/fargateService.go`: https://github.com/search?q=repo%3ADataDog%2Ftest-infra-definitions%20ECSFargateLinuxContainerDefinition&type=code